packages/context/test/acceptance/class-level-bindings.ts

```diff
@@ -171,6 +171,21 @@ describe('Context bindings - Injecting dependencies of classes', () => {
     expect(ctx.getSync('key')).to.equal('a-value');
   });
 
+  it('injects a nested property', async () => {
+    class TestComponent {
+      constructor(
+        @inject('config#test')
```

Comment:
Would it make sense here to use `@inject(`config${Binding.PropertySeparator}test`) here like in the other tests?

Reply:
@kjdelisle +1.

Reply:
This is an acceptance test that's should be as close to real world usage as possible. I don't expect application/extension developers to use `Binding.PropertySeparator` because it's way too verbose. Do you?

Reply:
No, I wouldn't rightly expect that. :P